.gitignore

```diff
@@ -17,3 +17,4 @@ coverage
 
 mainnet/**/outputs/*
 mainnet/outputs/*
+._*
```

Comment:
What's this one all about?

Reply:
macos pointer file generation when working on an external harddrive